index.d.ts

```diff
@@ -0,0 +1,24 @@
+export interface Options {
+	/**
+	 * Use a key distance-based score to leniently accept typos of `yes` and `no`.
+	 *
+	 * @default false
+	 */
+	lenient?: boolean;
+
+	/**
+	 * Default value if no match was found.
+	 *
+	 * @default null
+	 */
+	default?: boolean | null;
+}
+
+/**
+ * Parse yes/no like values.
+ * The following case-insensitive values are recognized: `'y', 'yes', 'true', true, '1', 1, 'n', 'no', 'false', false, '0', 0`
+ *
+ * @param input - Value that should be converted.
```

Comment:
Add `@param options - ...` as well or using method overloading.

```ts
export default function yn(input: any): boolean | null;

export default function yn(input: any, options: Options): boolean | null;
```

Reply:
For this case I followed the [typescript handbook do's & don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#use-optional-parameters)

Reply:
The @sindresorhus tsd style guide suggest [not adding options to docs](https://github.com/sindresorhus/typescript-definition-style-guide#documentation)

- If the parameter is `options` it doesn't need a description.

Reply:
Thanks for referring to the docs :).